documentation/index.ts

```diff
@@ -0,0 +1,497 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import j2s, { ComponentsSchema } from 'joi-to-swagger';
+import { Schema } from 'joi';
+import path from 'path';
+import fs from 'fs';
```

Comment:
Maybe I have different typescript options set than you do?

TS1192: Module '"fs"' has no default export.

Reply:
works for me